test/technology/charge_distribution_surface.cpp

```diff
@@ -15,41 +16,41 @@
-    (sidb_surface<
-        cell_level_layout<sidb_technology, clocked_layout<hexagonal_layout<siqad::coord_t, even_column_hex>>>>))
-{
-    REQUIRE(is_cell_level_layout_v<TestType>);
-    CHECK(!has_assign_charge_state_v<TestType>);
-    CHECK(!has_get_charge_state_v<TestType>);
-
-    TestType lyt{};
-
-    using charge_layout = charge_distribution_surface<TestType>;
-    CHECK(is_cell_level_layout_v<charge_layout>);
-    CHECK(has_assign_charge_state_v<charge_layout>);
-    CHECK(has_get_charge_state_v<charge_layout>);
-
-    const charge_layout defect_lyt{};
-    const charge_layout defect_lyt_from_lyt{lyt};
-
-    using charge_charge_layout = charge_distribution_surface<charge_layout>;
-    CHECK(is_cell_level_layout_v<charge_charge_layout>);
-    CHECK(has_assign_charge_state_v<charge_charge_layout>);
-    CHECK(has_get_charge_state_v<charge_charge_layout>);
-}
+// TEMPLATE_TEST_CASE(
+//     "Charge distribution surface traits and construction", "[charge-distribution-surface]",
+//     (cell_level_layout<sidb_technology, clocked_layout<cartesian_layout<siqad::coord_t>>>),
+//     (cell_level_layout<sidb_technology, clocked_layout<hexagonal_layout<siqad::coord_t, odd_row_hex>>>),
+//     (cell_level_layout<sidb_technology, clocked_layout<hexagonal_layout<siqad::coord_t, even_row_hex>>>),
+//     (cell_level_layout<sidb_technology, clocked_layout<hexagonal_layout<siqad::coord_t, odd_column_hex>>>),
+//     (cell_level_layout<sidb_technology, clocked_layout<hexagonal_layout<siqad::coord_t, even_column_hex>>>),
+//
+//     (sidb_surface<cell_level_layout<sidb_technology, clocked_layout<cartesian_layout<siqad::coord_t>>>>),
+//     (sidb_surface<cell_level_layout<sidb_technology, clocked_layout<hexagonal_layout<siqad::coord_t,
+//     odd_row_hex>>>>), (sidb_surface<cell_level_layout<sidb_technology,
+//     clocked_layout<hexagonal_layout<siqad::coord_t, even_row_hex>>>>), (sidb_surface<
+//         cell_level_layout<sidb_technology, clocked_layout<hexagonal_layout<siqad::coord_t, odd_column_hex>>>>),
+//     (sidb_surface<
+//         cell_level_layout<sidb_technology, clocked_layout<hexagonal_layout<siqad::coord_t, even_column_hex>>>>))
+//{
+//     REQUIRE(is_cell_level_layout_v<TestType>);
+//     CHECK(!has_assign_charge_state_v<TestType>);
+//     CHECK(!has_get_charge_state_v<TestType>);
+//
+//     TestType lyt{};
+//
+//     using charge_layout = charge_distribution_surface<TestType>;
+//     CHECK(is_cell_level_layout_v<charge_layout>);
+//     CHECK(has_assign_charge_state_v<charge_layout>);
+//     CHECK(has_get_charge_state_v<charge_layout>);
+//
+//     const charge_layout defect_lyt{};
+//     const charge_layout defect_lyt_from_lyt{lyt};
+//
+//     using charge_charge_layout = charge_distribution_surface<charge_layout>;
+//     CHECK(is_cell_level_layout_v<charge_charge_layout>);
+//     CHECK(has_assign_charge_state_v<charge_charge_layout>);
+//     CHECK(has_get_charge_state_v<charge_charge_layout>);
+// }
```

Comment:
What happened here?

Reply:
oh, sorry! I did this to find a bug but unfortunately, I forgot to undo it. Thanks you!